material tabindex imported incorrectly causing build failures downstream

Incorrect import `@angular/material/core/typings/common-behaviors/tabindex'` causing issues on downstream applications.



```bash
Module not found: Error: Can't resolve '@angular/material/core/typings/common-behaviors/tabindex' in 'C:\Users\yyy\projs\ief\ief-mobile\node_modules\ngx-cbp-theme'chunk {
inlineresolve '@angular/material/core/typings/common-behaviors/tabindex' in 'C:\Users\yy\projs\ief\ief-mobile\node_modules\ngx-cbp-theme'
} inline.bundle.js, inline.bundle.js.map  Parsed request is a module (inline) 5.83 kB 
[entry] [rendered]  using description file: C:\Users\yy\projs\ief\ief-mobile\node_modules\ngx-cbp-theme\package.json (relative path: .)
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -32,7 +32,3 @@ jobs:
         on:
           branch: master
 
-
-cache:
-  directories:
-    - node_modules

--- a/src/app/buttons/cbp-toggle-switch/cbp-toggle-switch.component.ts
+++ b/src/app/buttons/cbp-toggle-switch/cbp-toggle-switch.component.ts
@@ -7,7 +7,7 @@ import {
 } from '@angular/core';
 import {CanColor, CanDisable, mixinColor, mixinDisabled, mixinTabIndex, ThemePalette} from '@angular/material';
 import {ControlValueAccessor, NG_VALUE_ACCESSOR} from '@angular/forms';
-import {HasTabIndex} from '@angular/material/core/typings/common-behaviors/tabindex';
+import {HasTabIndex} from '@angular/material';
 
 
 export class CBPToggleSwitchChange {

--- a/src/kitchensink/app/demo-buttons/demo-buttons.component.html
+++ b/src/kitchensink/app/demo-buttons/demo-buttons.component.html
@@ -18,39 +18,39 @@
 <form>
     <div>
         <cbp-toggle-switch label="Access" name="toggle-one" [(ngModel)]="toggle.one"
-                           (changed)="toggle.one = $event"></cbp-toggle-switch>
+                           (change)="changed($event)"></cbp-toggle-switch>
         <span style="padding: 10px;">value = {{ toggle.one }}</span>
     </div>
     <br>
     <div>
         <cbp-toggle-switch name="toggle-something2" label="Bidi Model" onLabel="YES" offLabel="NO"  name="toggle-something3" [(ngModel)]="toggle.something2"
-                           (changed)="changed($event)"></cbp-toggle-switch>
+                           (change)="changed($event)"></cbp-toggle-switch>
         <span style="padding: 10px;min-width: 100px;display: inline-block;">value = {{ toggle.something2 }}</span>
         <mat-checkbox [disableRipple]="true" name="toggle-something2-checkbox" [(ngModel)]="toggle.something2">Toggle</mat-checkbox>
     </div>
     <br>
     <div>
         <cbp-toggle-switch label="Alert" name="toggle-two" onLabel="YES" offLabel="NO"
-                           [(ngModel)]="toggle.two" (changed)="changed($event)"></cbp-toggle-switch>
+                           [(ngModel)]="toggle.two" (change)="changed($event)"></cbp-toggle-switch>
         <span style="padding: 10px; width: 50px;">value = {{ toggle.two }}</span>
     </div>
     <br>
     <div>
         <cbp-toggle-switch label="Y/N Model values" name="toggle-yn" onLabel="YES" offLabel="NO" onValue="Y" offValue="N"
-                           [(ngModel)]="toggle.yn" (changed)="changed($event)"></cbp-toggle-switch>
+                           [(ngModel)]="toggle.yn" (change)="changed($event)"></cbp-toggle-switch>
         <span style="padding: 10px; width: 50px;">value = {{ toggle.yn }}</span>
     </div>
     <br>
     <div>
         <cbp-toggle-switch label="Undefined" name="toggle-three" onLabel="YES" offLabel="NO"
-                           [(ngModel)]="toggle.three" (changed)="changed($event)"></cbp-toggle-switch>
+                           [(ngModel)]="toggle.three" (change)="changed($event)"></cbp-toggle-switch>
         <span style="padding: 10px; width: 50px;">value = {{ toggle.three }}</span>
     </div>
     <br>
     <div>
         <cbp-toggle-switch label="Disabled" disabled="true" onLabel="YES" offLabel="NO" onValue="Y" offValue="N"
                            name="toggle-disabled" [(ngModel)]="toggle.disabled"
-                           (changed)="toggle.disabled = $event"></cbp-toggle-switch>
+                           (change)="changed($event)"></cbp-toggle-switch>
         <span style="padding: 10px;">value = {{ toggle.disabled }}</span>
     </div>
     <br>

